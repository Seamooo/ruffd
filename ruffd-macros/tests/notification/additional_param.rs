use ruffd_macros::notification;

#[notification]
async fn some_notification(params: i32, bad_param: i32) -> Result<(), ruffd_types::RpcError> {
    Ok(())
} 

fn main() {}

