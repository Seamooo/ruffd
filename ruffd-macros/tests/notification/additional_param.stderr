error: Exactly one or zero parameters allowed
 --> tests/notification/additional_param.rs:3:1
  |
3 | #[notification]
  | ^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `notification` (in Nightly builds, run with -Z macro-backtrace for more info)
